### ISSUE TITLE
Installation task

### ISSUE THREAD

**Issue by RyodoTanaka** [OUXT-Polaris/vrx_tasks#2]

**Comment by hakuturu583** [comment]
VRX source code is here.
https://bitbucket.org/osrf/vrx/src/default/

**Comment by hakuturu583** [comment]
Wiki page including install instruction is here.
https://bitbucket.org/osrf/vrx/wiki/Home

**Comment by RyodoTanaka** [comment]
# Using docker
### 1. Install docker
### 2. Install nvidia-docker2
### 3. Build VRX Dockerfile
https://bitbucket.org/osrf/vrx/wiki/tutorials/buildRunLocalImage

If first launch file does not work, try to remove `nvidia-docker2` and install `nvidia-docker`.

## Problem
After above installation, I tried to install CUDA10.* and CUDN7.5.* .
```bash
$ wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-repo-ubuntu1804_10.0.130-1_amd64.deb
$ sudo dpkg -i cuda-repo-ubuntu1804_10.0.130-1_amd64.deb
$ sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub
$ sudo apt update
$ sudo apt install cuda
```
But I got following error message.
```log
Errors were encountered while processing:
 /tmp/apt-dpkg-install-hb8UrJ/070-nvidia-compute-utils-418_418.67-0ubuntu1_amd64.deb
 /tmp/apt-dpkg-install-hb8UrJ/075-nvidia-utils-418_418.67-0ubuntu1_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```
And `nvidia-smi` returns following.
```log
Failed to initialize NVML: Driver/library version mismatch
```

Here, I am stacking....

**Comment by RyodoTanaka** [comment]
# What I did
### Try to install CUDA10.0 + OpenGL 1.0 docker image
https://hub.docker.com/r/nvidia/cudagl
Based on above image, I tried to install vrx docker image.
But, I got following error when I run the vrx image with `run.bash` script.
```log
docker: Error response from daemon: OCI runtime create failed: container_linux.go:345: starting container process caused "process_linux.go:424: container init caused \"process_linux.go:407: running prestart hook 1 caused \\\"error running hook: exit status 1, stdout: , stderr: exec command: [/usr/bin/nvidia-container-cli --load-kmods configure --ldconfig=@/sbin/ldconfig.real --device=all --compute --utility --graphics --compat32 --utility --require=cuda>=10.0 brand=tesla,driver>=384,driver<385 brand=tesla,driver>=410,driver<411 --pid=18508 /var/lib/docker/overlay2/dbcb07f1cf9db2f25623dbb54fa7f028a979693a019103e64def073763e8e0a4/merged]\\\\nnvidia-container-cli: requirement error: unsatisfied condition: brand = tesla\\\\n\\\"\"": unknown.
```
### Reason
I found following issue.
https://github.com/NVIDIA/nvidia-docker/issues/832#issuecomment-425605760

My GPU is Geforce GTX 560.
This GPU hardware version is Fermi. That is shown on following link.
https://en.wikipedia.org/wiki/List_of_Nvidia_graphics_processing_units#GeForce_500_series

And here is the CUDA/driver available version table.
https://docs.nvidia.com/deploy/cuda-compatibility/index.html

From above table, the Fermi hardware version GPU is only able until CUDA 9.0.
So, I need new GPU !!!!

**Comment by RyodoTanaka** [comment]
# Fix to install OpenGL 1.0 + CUDA 10.1 environment

### 1. Install nvidia driver later than 418.39 .
### 2. Build it

**Comment by RyodoTanaka** [comment]
I created the Docker image and upload it on Docker Hub.
https://cloud.docker.com/u/ouxt/repository/docker/ouxt/vrx_packages
